LiveSPICEVst/EditorView.xaml.cs

```diff
@@ -3,6 +3,7 @@
 using System.IO;
 using System.Windows;
 using System.Windows.Controls;
+using static System.Windows.Forms.VisualStyles.VisualStyleElement.Window;
```

Comment:
Is this actually used? May have been an accidental intellisense addition? (That happens to me a lot.)

Reply:
You are absolutely right. I installed Visual Studio just to open this project and since I haven't used it in well over 5-6 years, I forgot to check the intellisense additions, which is my bad. I removed the unnecessary line now and everything should be as requested. If you have any other comments, let me know!